uplyft/models.py

```diff
@@ -57,7 +56,7 @@ def __str__(self):
 def file_size(value):
     limit = 2 * 1024 * 1024
     if value.size > limit:
-        raise ValidationError("File too large. Size should not exceed 2 MB")
+        raise ValidationError("File too large. Size should not exceed 2 MiB")
```

Comment:
@ahanif16 same question about MiB

Reply:
Same answer as above. Our current size limit is in fact 2 MiB but we can update to 2MB